recipes/absent/all/conanfile.py

```diff
@@ -19,36 +19,29 @@ class AbsentConan(ConanFile):
     def _source_subfolder(self):
         return "source_subfolder"
 
-    @property
-    def _build_subfolder(self):
-        return "build_subfolder"
```

Comment:
Why did you remove build_subfolder? The propose here is keeping all build stuff separated from the source files.

Reply:
Because the library is header-only so I thought the build step and things related wouldn't be needed.

However, there's a WARN in the logs about not having the build step defined.

What should be the recommended way? Should I bring the build step back?

Reply:
If it doesn't require a build step, it doesn't require it 👍  maybe we should show that warning in Conan.

Reply:
Got it.
Thanks for clarifying that.